CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # avro-patches
 
+## v0.3.0
```

Comment:
It doesn't matter much but out of curiosity could this be a maintenance version bump?